saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/property/features/real/RealGenProperty.scala

```diff
@@ -31,7 +28,7 @@ case class RealGenProperty[T <: AnyRef](
       val d: T = example.asInstanceOf[T]
       val values = sensor(d)
 
-      var featureVector = new FeatureVector
+      val featureVector = new FeatureVector
```

Comment:
Just asking, do we have tests that if the FeatureVector update is not correct we realize it? Sometimes these changes can make semantic errors, no?